changelog.d/10561.bugfix

```diff
@@ -0,0 +1 @@
+Handle failure of User-Interactive Authentication when using a fallback. Contributed by Callum Brown.
```

Comment:
Is this handling the failure or displaying an error? It seems it was previously "handled", but by displaying a JSON error message to the user?